MIGRATION.md

```diff
@@ -1,7 +1,8 @@
 <h1>Migration</h1>
 
 - [From version 6.0.x to 6.1.0](#from-version-60x-to-610)
-  - [6.1 deprecations](#61-deprecations)
+  - [6.1 deprecations](#6dot1-deprecations)
```

Comment:
Does this work?

Reply:
No hmm seems like my vscode broke suddenly 😦  Good catch